.lintstagedrc.mjs

```diff
@@ -1,6 +1,5 @@
 export default {
     '*': 'prettier --write --ignore-unknown',
-    'packages/**/*.{ts,tsx,js,jsx}': 'eslint -c packages/.eslintrc.json --fix',
```

Comment:
？

Reply:
It was old commit, just to push code on my fork, it is fine now.